rotkehlchen/accounting/structures/base.py

```diff
@@ -119,22 +119,22 @@ def __init__(
         self.notes = notes
         self.identifier = identifier
 
-    def __eq__(self, other: Any) -> bool:
+    def __eq__(self, other: object) -> bool:
```

Comment:
shouldn't the type here be

```suggestion
    def __eq__(self, other: HistoryBaseEntry) -> bool:
```

also similar thing in the places where the same pattern has changed

Reply:
I believe the right type is object since you have to compare against anything but inside what is used are properties of HistoryBaseEntry that will fail if `other` is not of a subtype of `HistoryBaseEntry`